packages/integration-sdk-http-client/src/client.ts

```diff
@@ -322,9 +297,11 @@ export abstract class BaseAPIClient {
             } else {
               error = await fatalRequestError(requestErrorParams);
             }
-            for await (const _chunk of response.body) {
-              // force consumption of body to avoid memory leaks
-              // https://github.com/node-fetch/node-fetch/issues/83
+            if (response.body) {
+              for await (const _chunk of response.body) {
+                // force consumption of body to avoid memory leaks
+                // https://github.com/node-fetch/node-fetch/issues/83
```

Comment:
I can't wait until we are on node 21 and we can start using Global Fetch.

Reply:
Yes! Me too